db/migrate/20160322224507_make_foreign_key_columns_integers.rb

```diff
@@ -1,6 +0,0 @@
-class MakeForeignKeyColumnsIntegers < ActiveRecord::Migration
-  def change
-    change_column :payment_braintree_customers, :default_payment_method_id, 'integer USING CAST(default_payment_method_id AS integer)'
-    change_column :payment_braintree_transactions, :payment_method_id, 'integer USING CAST(payment_method_id AS integer)'
-  end
-end
```

Comment:
I killed this because `default_payment_method_id` isn't required anymore (as per above), and `payment_method_id` is created as a string by braintree ( eg '3d4k3s')